react-app/package.json

```diff
@@ -4,8 +4,9 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.6.5",
-    "@cosmjs/proto-signing": "^0.28.4",
-    "@cosmjs/stargate": "^0.28.4",
+    "@cosmjs/proto-signing": "^0.27.1",
+    "@cosmjs/stargate": "^0.27.1",
+    "@desmoslabs/desmjs": "^2.3.1-beta.0",
```

Comment:
have you tried using desmjs `^3.0.0-alpha.2`? that seems to be the latest and working version for me

Reply:
Yes, but `desmjs` `v3.0.0` uses `desmos` `v3.0.0`, but on mainnet it seems `desmos` is still on `v2.3.1`

See https://github.com/desmos-labs/desmjs/tree/main#compatibility-table

Reply:
oh sad